### PR TITLE
Jake/sc 157812/setup auto format

### PR DESCRIPTION
**Pull request by jakegraves-figure**

### CODE REVIEW

**Review by dhancockFigure — APPROVED** [review]

File: .git-blame-ignore-revs [hunk 1]
```diff
@@ -0,0 +1 @@
+# .git-blame-ignore-revs
```
Comment (royd) [inline .git-blame-ignore-revs:1]: Could add the formatting commit hash here

File: README.md [hunk 2]
```diff
@@ -2,3 +2,36 @@
 The home of all shared Dart resources for the Provenance ecosystem.
 
 ![Build](https://github.com/provenance-io/provenance-dart/actions/workflows/build.yml/badge.svg)
+
+## Getting Started
```
Comment (royd) [inline README.md:6]: 👍

**Review by royd — APPROVED** [review]
I could go either way on formatting the generated code or specifically excluding it. 

Would it be nice to have the generated code in a `generated` dir?

**Comment by jakegraves-figure** [comment]
> I could go either way on formatting the generated code or specifically excluding it.
> 
> Would it be nice to have the generated code in a `generated` dir?

I like this, but it probably needs to go in another pull request.

**Pull request merged**